### DISCUSSION TITLE
Reduce CPU/Memory usage when no players are connected

### DISCUSSION THREAD

**Discussion by knaak** [lloesche/valheim-server-docker#350] [Q&A] [answered]
I've noticed that this software uses about 25% of my CPU pretty continuously even when no players are connected.  Although my friends and I played Valheim very frequently, after we beat the last boss our play time is reduced.  I keep the server running so that anyone can connect in when they wish to which is fairly rare at the moment.

Any suggestions on how to reduce resource consumption when there are no active connections?  I see that I can reduce backup frequency, but is there anything else that can be done or is that restricted by the underlying software that has been containerized?

Tim

**Comment by lloesche** [comment, marked as answer]
> restricted by the underlying software that has been containerized

this. The valheim dedicated server is what consumes all that CPU. The miscellaneous scripts like backup and updater that come with the container use basically no CPU cycles at all compared to the dedicated server itself.
